tests/Feature/ImportExport/UtilsTest.php

```diff
@@ -31,4 +32,61 @@ public function testGetPmConfig()
         $config = Utils::getPmConfig($result->first());
         $this->assertEquals('list', $config['endpoint']);
     }
+
+    public function testFindScreenDependent()
+    {
+        $config = [
```

Comment:
maybe this item can be a json in the "fixtures" folder

Reply:
Good idea, fixed